src/main/scala/dpla/ingestion3/model/package.scala

```diff
@@ -57,6 +57,8 @@ package object model {
 
   def emptyEdmWebResource: EdmWebResource = stringOnlyWebResource("")
 
+  def emptyJValue: JValue = "" -> ""
```

Comment:
Wondering out loud: Would JNothing or JNull work here?

Reply:
No, you are right, that would work